adafruit_bitmapsaver.py

```diff
@@ -0,0 +1,145 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2019 Dave Astels for Adafruit Industries
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+"""
+`adafruit_bitmapsaver`
+================================================================================
+
+Save a displayio.Bitmap (and associated displayio.Palette) in a BMP file.
+Make a screenshot (the contents of a displayio.Display) and save in a BMP file.
+
+
+* Author(s): Dave Astels
+
+Implementation Notes
+--------------------
+
+**Hardware:**
+
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+"""
+
+# imports
+
+import gc
+import struct
+import board
+from displayio import Bitmap, Palette, Display
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_BitmapSaver.git"
+
+#pylint:disable=line-too-long,broad-except,redefined-outer-name,invalid-name
```

Comment:
hiya don't disable these, they're there for a reason! please lint the code so these errors don't come up.